upstart/ua-premium.conf

```diff
@@ -0,0 +1,14 @@
+# ua-premium - Ubuntu Advantage Premium attach
+
+# This task is run on reboot to adapt to potential change in
+# kernels/environment across boot and update Ubuntu Advantage status
+# information.
+
+description    "Ubuntu Advantage Premium image attach"
+
+start on runlevel [2345]
+
+task
+# Refresh the status of all Ubuntu Advantage services
```

Comment:
this comment doesn't seem to do what you say. We are attaching, not refreshing the status.

Reply:
reworded.